### ISSUE TITLE
[FSR] Update green buttons to be action links

### ISSUE THREAD

**Issue by rebeccawalsh** [department-of-veterans-affairs/va.gov-team#23287]
## Issue Description

Per [design feedback from VSP](https://github.com/department-of-veterans-affairs/va.gov-team/issues/22943):
Green buttons are being removed. Replace our 3 instances of green buttons with the new [action link component](https://design.va.gov/components/action-links) on the [wizard](https://preview.uxpin.com/803dc24213cf723bfce6c5e3eaef5a2c3f3e344b#/pages/133701647/specification/sitemap?mode=i) and introduction ([auth](https://preview.uxpin.com/803dc24213cf723bfce6c5e3eaef5a2c3f3e344b#/pages/133701378/specification/sitemap?mode=i) and [unauth](https://preview.uxpin.com/803dc24213cf723bfce6c5e3eaef5a2c3f3e344b#/pages/133701377/specification/sitemap?mode=i)) pages 

---
## Tasks
- [ ] Swap out buttons for action links

## Acceptance Criteria
- [ ] Buttons swapped!

**Comment by rebeccawalsh** [comment]
Switched this from 'blocker' to blocked, since I heard from Shawna that this change wasn't tested by VSP, and when these links were used on the Caregiver form usability study, they tested very poorly in the way we've been told to use them.

[Design ticket](https://app.zenhub.com/workspaces/vft-59c95ae5fda7577a9b3184f8/issues/department-of-veterans-affairs/va.gov-team/23311)

**Comment by LinnJS** [comment]
I can pick this up since we just also did this on the 10-10CG and other teams are rolling it out. Is it okay if I unblock it @rebeccawalsh ?

**Comment by Tom-Davis** [comment]
This ticket is not needed for the approach being taken.
